core/src/main/java/org/jivesoftware/sparkimpl/plugin/transcripts/HistoryTranscript.java

```diff
@@ -262,7 +260,8 @@ private synchronized void display() {
 			token.acquire();
 
 			if ((searchFilteredList.size() > 0) && (pageIndex <= searchFilteredList.size())) {
-				builder.append(buildString(searchFilteredList.get(pageIndex-1).getMessages()));
+                builder.append(org.apache.commons.lang3.StringEscapeUtils.unescapeHtml4(
```

Comment:
Could you use an import statement? That way, you can do ``StringEscapeUtils.unescapeHtml4`` instead of ``org.apache.commons.lang3.StringEscapeUtils.unescapeHtml4`` - which makes the code more readable.